.github/workflows/main.yml

```
@@ -0,0 +1,55 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '--color=yes'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python: ["2.7", "3.6", "3.7"]
```

I have a slight preference for list style since it's easier to add/remove a single entry since the diff is easier to read.
```suggestion
        python:
          - "2.7"
          - "3.6"
          - "3.7"
```

Perhaps also add 3.8?